src/main/cpp/DragonVision/DragonVision.h

```diff
@@ -18,60 +18,85 @@
 #include <map>
 #include <string>
 
-#include "State.h"
-#include "DragonVision/DragonCamera.h"
+// FRC Includes
+#include "frc/apriltag/AprilTagFieldLayout.h"
+#include "frc/apriltag/AprilTagFields.h"
 
-using std::map;
+#include "DragonVision/DragonVisionStructs.h"
+#include "DragonVision/DragonCamera.h"
 
-class DragonLimelight;
+class DragonCamera;
 class DragonVision
 {
 public:
     static DragonVision *GetDragonVision();
 
+    static frc::AprilTagFieldLayout GetAprilTagLayout();
+
     enum CAMERA_POSITION
     {
-        FRONT,
-        BACK,
-        BACK_INTAKE
+        LAUNCHER,
+        PLACER,
+        PLACER_INTAKE,
+        LAUNCHER_INTAKE
     };
 
     enum VISION_ELEMENT
     {
+        LAUNCHER_NOTE,
+        PLACER_NOTE,
         NOTE,
-        RED_SPEAKER,
-        BLUE_SPEAKER,
-        RED_SUBWOOFER,
-        BLUE_SUBWOOFER,
-        RED_AMP,
-        BLUE_AMP,
-        RED_CENTER_STAGE,
-        RED_LEFT_STAGE,
-        RED_RIGHT_STAGE,
-        BLUE_CENTER_STAGE,
-        BLUE_LEFT_STAGE,
-        BLUE_RIGHT_STAGE
+        SPEAKER,
+        AMP,
+        STAGE,
```

Comment:
I think we need Left, Right and Center stage distinctions

Reply:
I talked with Tanay about that.  The current plan is to have this more as a filter on GetDataToNearestAprilTag by filtering the tag id's we return data from to only be the stage tags.  If you disagree, we can add those distinctions and do the same thing as the Notes where we can specify which one we want, or just have a general "whichever is easiest" option.